shared-bindings/vectorio/Circle.c

```diff
@@ -81,6 +81,21 @@ const mp_obj_property_t vectorio_circle_radius_obj = {
 };
 
 
+// Documentation for properties inherited from VectorShape.
+
+//|     x : int
+//|     """X position of the center point of the circle in the parent."""
+//|
+//|     y : int
+//|     """Y position of the center point of the circle in the parent."""
+//|
+//|     location : Tuple[int,int]
+//|     """(X,Y) position of the center point of the circle in the parent."""
+//|
+//|     pixel_shader : Union[displayio.ColorConverter,displayio.Palette]
+//|     """The pixel shader of the circle."""
+//|
```

Comment:
Why repeat these? Definition for these properties is already available in the parameters section.

Reply:
It makes it clearer that they are properties and be `get` and `set` after the object is constructed.
Before
![Screenshot from 2021-09-12 12-56-23](https://user-images.githubusercontent.com/70200140/132986745-cb7f523a-f241-41c6-9f9d-996c03f75372.png)
After
![Screenshot from 2021-09-12 13-01-04](https://user-images.githubusercontent.com/70200140/132986747-edf9585f-6eca-461b-9291-d7bcb3abab33.png)